tap_twilio/streams.py

```diff
@@ -431,8 +431,7 @@
                         'path': 'Workspaces/{ParentId}/Workers/{ParentId}/Channels',
                         'data_key': 'channels',
                         'key_properties': ["sid"],
-                        'replication_method': 'INCREMENTAL',
-                        'replication_keys': ['date_updated'],
+                        'replication_method': 'FULL_TABLE',
```

Comment:
how many rows we expect in the worker `channels` table? I'm just wondering if there is any implications of using `FULL_TABLE`.

Reply:
We have 1 Workspace, 400ish CS agents (=Worker) today, 600 later this year, and 5 Channel per agent.
So right now it's 2000 Channels.
End of year it will be 3000 Channels.